i18n-helpers/src/lib.rs

```diff
@@ -569,12 +569,22 @@ fn check_code_block_token_count(group: &[(usize, Event)]) -> usize {
     let events = group.iter().map(|(_, event)| event);
     let mut in_codeblock = false;
     let mut max_token_count = 0;
+
+    // token_count should be taken over Text events
+    // because a continuous text may be splitted to some Text events.
```

Comment:
Nit: If this is about splitting a single text into several `Text` events, then `several` instead of `some` might be more accurate in this context.